src/viewer.js

```diff
@@ -41,6 +42,25 @@ class Viewer {
 
     /** @private {?Element} */
     this.iframe_ = null;
+
+    /** @private {!History} */
+    this.history_ = new History(this.unAttach.bind(this));
+  }
+
+  /**
+   * @param {!Function} hideViewer method that hides the viewer.
+   */
+  setViewerHider(hideViewer) {
+    /** @private {!Function} */
+    this.hideViewer_ = hideViewer;
+  }
+
+  /**
+   * Enables history Fragment for the init params.
+   */
+  enableHistoryFragment() {
```

Comment:
Up to you, but you could also pass in a config object. For example:
http://jsbin.com/qarohaj/edit?html,js,console

That way you can still declare fields as const and don't need to worry about unexpected changes.